src/main.rs

```diff
@@ -895,114 +950,15 @@ pub async fn verify_from_repo(
     current_dir: bool,
     skip_prompt: bool,
     path_to_keypair: Option<String>,
+    compute_unit_price: u64,
+    mut skip_build: bool,
     container_id_opt: &mut Option<String>,
     temp_dir_opt: &mut Option<String>,
+    check_signal: &dyn Fn(&mut Option<String>, &mut Option<String>),
 ) -> anyhow::Result<()> {
-    if remote {
-        let genesis_hash = get_genesis_hash(connection)?;
-        if genesis_hash != MAINNET_GENESIS_HASH {
-            return Err(anyhow!("Remote verification only works with mainnet. Please omit the --remote flag to verify locally."));
-        }
```

Comment:
Why is this entire block getting deleted?

Reply:
It was duplicate code exactly the same as 100 lines below. Both code paths now use the same args logic

Reply:
I think you forgot to add this get_genesis_hash check back in

Reply:
Its still there, no? In line 1140

Reply:
You're right! Looks good to me